python/ray/autoscaler/aws/cloudwatch/prometheus_ray_up.sh

```diff
@@ -0,0 +1,35 @@
+###########################################################################################################
+#This bash script is meant to support prometheus metrics collection via cloudwatch agent.
+#You can view the ray metrics being collected through AWS CloudWatch console.
+###########################################################################################################
+
+read -r -p $'Usage: prometheus_ray_up.sh [OPTIONS] CLUSTER_CONFIG_FILE:\n' RAY_UP_OPTIONS
+AUTOSCALER_CONFIG_FILE=$(sed 's/.* //' <<< "$RAY_UP_OPTIONS")
+CLUSTER_NAME=$(grep 'cluster_name' $AUTOSCALER_CONFIG_FILE| cut -d ':' -f 2 | xargs)
+
+# ray up to start a new cluster
+ray up $RAY_UP_OPTIONS
+
+if [ $? -eq 0 ]; then
+  # TODO remove the version number hardcoding from this script
+  # download the latest prometheus
+  ray exec "$AUTOSCALER_CONFIG_FILE" "sudo wget https://github.com/prometheus/prometheus/releases/download/v2.25.0/prometheus-2.25.0.linux-amd64.tar.gz -P /opt"
+  ray exec "$AUTOSCALER_CONFIG_FILE" "cd /opt && sudo tar xvfz prometheus-2.25.0.linux-amd64.tar.gz"
+  echo "Successfully installed prometheus on head node"
+
+  # replace the default prometheus server configuration with ray services discovery supported configuration
+  ray exec "$AUTOSCALER_CONFIG_FILE" "sudo cp --remove-destination /tmp/prometheus.yml /opt/prometheus-2.25.0.linux-amd64"
+  echo "Successfully configured prometheus on head node"
+
+  # start the prometheus server
+  ray exec "$AUTOSCALER_CONFIG_FILE" "cd /opt/prometheus-2.25.0.linux-amd64 && sudo ./prometheus --config.file=prometheus.yml --web.enable-lifecycle --log.level=info &"
+  echo "Successfully started prometheus on head node"
+
+  # restart the cloudwatch agent with prometheus integrated
+  echo "Restarting the cloudwatch agent to enable prometheus integration...This may take a few minutes."
```

Comment:
Minor: I think we can shorten this message to simply, "Restarting cloudwatch agent... this may take a few minutes."